test/test-functions

```diff
@@ -99,11 +103,18 @@ run_qemu() {
         exit 1
     fi
 
-    KERNEL_APPEND="root=/dev/sda1 \
+if [[ "$LOOKS_LIKE_SUSE" ]]; then
+    PARAMS+="systemd.mask=systemd-networkd.service "
+    PARAMS+="rd.hostonly=0"
```

Comment:
Thanks for the hint, indeed it should be assured that the initrd contains the systemd version that is to be tested. Currently we just install this systemd version on the host before running the tests. But it seems that the code for creating a dedicated test initrd hasn't been implemented yet. So until it is there, I'd prever to keep the above parameters, since they have been proven to work on SUSE.